### PR TITLE
Implementation of translation system in the site

### PR DESCRIPTION
**Pull request by ln-dev7**
This PR related #39 

@MedouneSGB @daoodaba975 @NDOY3M4N

### CODE REVIEW

File: src/pages/components/Navbar.tsx [hunk 1]
```diff
@@ -1,24 +1,40 @@
-import { useState, Fragment, useEffect } from 'react';
-import { Dialog, Transition } from '@headlessui/react';
+import React, { useState, Fragment, useEffect } from 'react';
```
Comment (NDOY3M4N) [inline src/pages/components/Navbar.tsx:1]: Do we need the `React` import at the top of the file?

**Review by NDOY3M4N — APPROVED** [review]
Wow ❇️ @ln-dev7 thanks for this huge contribution, we really needed this plugin for the website.

File: src/pages/components/Navbar.tsx (hunk 1 above)
Comment (ln-dev7) [reply src/pages/components/Navbar.tsx:1]: No

**Comment by daoodaba975** [comment]
Nice job @ln-dev7 appreciate the effort 😎🚀

**Pull request merged**